docs/markdown/snippets/meson_options.md

```diff
@@ -0,0 +1,7 @@
+## Support for reading options from meson.options
+
+Support has been added for reading options from `meson.options` instead of
+`meson_options.txt`. These are equivalent, but not using the `.txt` extension
+for a build file has a few advantages, chief among them many tools and exte
+eidtors expect a file with the `.txt` extension to be plain text files, not
```

Comment:
Presumably this should be "text editors" rather than "exte eidtors". :)

Reply:
lol, yes